website/src/pages/StatusPage/components/ExecutionQueue/ExecutionsQueueTableToolbar.tsx

```diff
@@ -0,0 +1,135 @@
+/*
+Copyright 2024 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { Cross2Icon } from "@radix-ui/react-icons";
+import { Table } from "@tanstack/react-table";
+
+import { Button } from "@/components/ui/button";
+import { DataTableViewOptions } from "@/components/ui/data-table-view-options";
+import { Input } from "@/components/ui/input";
+
+import { DataTableFacetedFilter } from "@/components/ui/data-table-faceted-filter";
+
+const workloadses = [
+  {
+    label: "oltp",
+    value: "oltp",
+  },
+  {
+    label: "oltp-readonly",
+    value: "oltp-readonly",
+  },
+  {
+    label: "oltp-set",
+    value: "oltp-set",
+  },
+  {
+    label: "tpcc",
+    value: "tpcc",
+  },
+  {
+    label: "tpcc_fk",
+    value: "tpcc_fk",
+  },
+  {
+    label: "tpcc_fk_unmanaged",
+    value: "tpcc_fk_unmanaged",
+  },
+  {
+    label: "tpcc_unsharded",
+    value: "tpcc_unsharded",
+  },
+];
+const sourceses = [
+  {
+    label: "cron",
+    value: "cron",
+  },
+  {
+    label: "cron_pr",
+    value: "cron_pr",
+  },
+  {
+    label: "cron_pr_base",
+    value: "cron_pr_base",
+  },
+  {
+    label: "cron_tags",
+    value: "cron_tags",
+  },
+];
```

Comment:
We should not hardcode this two constants. We should base them on what the API is returning I think. There can be an infinite number of new sources, and the list of workload can change from time to time.

Reply:
Should we create new endpoints that returns the list of possible workloads and sources ?